### ISSUE TITLE
Can we add more description to List Of Supported Sites?

### ISSUE THREAD

**Issue by J2TeamNNL** [sherlock-project/sherlock#1785]
I think the more detail info about sites, the more we know we are searching for. Cause none knows all of that list about.

[List Of Supported Sites](https://github.com/sherlock-project/sherlock/blob/master/sites.md)

**Comment by LucianCrainic** [comment]
I agree, there should be a basic template for each website where we can see some information about it.

**Comment by ngv-sama** [comment]
Hi! I am new to open source, could you please explain the issue? I'd like to work on it.

**Comment by KingWilliamWC** [comment]
We could categorise the list into different categories? (Entertainment, Music, Programming, NSFW)
